pkgs/shells/rc-9front/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchgit
+, byacc
+, installShellFiles
+}:
+
+stdenv.mkDerivation {
+  pname = "rc-9front";
+  version = "unstable-2022-11-01";
+
+  src = fetchgit {
+    url = "git://shithub.us/cinap_lenrek/rc";
+    rev = "69041639483e16392e3013491fcb382efd2b9374";
+    hash = "sha256-xc+EfC4bc9ZA97jCQ6CGCzeLGf+Hx3/syl090/x4ew4=";
+  };
+
+  strictDeps = true;
+  nativeBuildInputs = [ byacc installShellFiles ];
+  enableParallelBuilding = true;
+  patches = [ ./path.patch ];
+
+  buildPhase = ''
+    make PREFIX=$out
```

Comment:
Hum, it should be not needed. A simple `makeFlags = [prefix=out]` should suffice.